src/doc/dev_manual/Spack.rst

```diff
@@ -0,0 +1,362 @@
+Building with Spack
+===================
+
+Overview
+--------
+
+Spack is a multi-platform package manager that builds and installs multiple versions and configurations of software. It works on Linux, macOS, and many supercomputers. Spack is non-destructive: installing a new version of a package does not break existing installations, so many configurations of the same package can coexist.
+
+Spack offers a simple "spec" syntax that allows users to specify versions and configuration options. Package files are written in pure Python, and specs allow package authors to write a single script for many different builds of the same package. With Spack, you can build your software all the ways you want to.
+
+The complete documentation can be found `here <https://spack.readthedocs.io/en/latest/index.html>`__.
+
+The VisIt_ spack package
+------------------------
+
+The spack package for VisIt_ currently only builds a subset of the libraries that build_visit builds.
```

Comment:
Best if we indicate `build_visit` is a command/tool by typesetting it as such (surround with double-back-ticks). Again, maybe something to audit throughout.